Swift-Base/AppDelegate.swift

```diff
@@ -29,6 +29,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         #endif
         
+        #if DEBUG
+            // print documents directory and device ID
+            print("\n*******************************************\nDOCUMENTS\n\(NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true)[0])\n*******************************************\n")
```

Comment:
@VladimirGoncharov I will create ```print(someText) ``` function in FSHelpers today. And then I will replace this functions.
But I would like to use 
```
#if DEBUG
print(someText)
#endif
```

because in won't add to source while compiling for release.